apps/wallet/src/ui/app/pages/home/tokens/SupplyIncreaseVestingStakingDialog.tsx

```diff
@@ -31,16 +31,50 @@ export function SupplyIncreaseVestingStakingDialog({
             <DialogContent containerId="overlay-portal-container">
                 <Header title="Vested Staking" onClose={() => setOpen(false)} titleCentered />
```

Comment:
Now that we are requiring an action from the user, i wouldnt call it vested staking but rather "Vesting" everywhere, so its called the same as in the dashboard
```suggestion
                <Header title="Vesting" onClose={() => setOpen(false)} titleCentered />
```